rest-api/config/config_prod.json

```diff
@@ -124,6 +124,19 @@
       "whitelisted_ip_ranges" : {
         "ip4": ["160.129.251.78"]
       }
+    },
+    "awardee-newyork@all-of-us-ops-data-api-prod.iam.gserviceaccount.com": {
+      "roles": ["awardee_sa"],
+      "awardee": "newyork",
```

Comment:
I believe this should be columbia.

Reply:
Are you sure? I saw there is another account for columbia is "awardee-columbia@all-of-us-ops-data-api-stable.iam.gserviceaccount.com". And in the ticket PD-2165, Celecia said it's "newyork".

Reply:
we use column A of this sheet https://docs.google.com/spreadsheets/d/1CcIGRV0Bd6BIz7PeuvrV6QDGDRQkp83CJUkWAl-fG58/edit#gid=1076878570
There is no "new york" awardee ID. 
We have a columbia because another organization under the awardee has already signed up.
We can have duplicates here, the `awardee` key is used to ensure the ops-data api doesn't search for other awardees.

Reply:
done